sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -160,23 +170,55 @@ private LocalDocumentStorage(DatabaseManager databaseManager) {
         return new Document<>(new StorageException("Document was not found in the cache."));
     }
 
+    private static SQLiteQueryBuilder getPartitionAndDocumentIdQueryBuilder() {
+        SQLiteQueryBuilder builder = SQLiteUtils.newSQLiteQueryBuilder();
+        builder.appendWhere(BY_PARTITION_AND_DOCUMENT_ID_WHERE_CLAUSE);
+        return builder;
+    }
+
+    private static ContentValues getContentValues(PendingOperation operation, long now) {
+        ContentValues values = new ContentValues();
+        values.put(PARTITION_COLUMN_NAME, operation.getPartition());
+        values.put(DOCUMENT_ID_COLUMN_NAME, operation.getDocumentId());
+        values.put(DOCUMENT_COLUMN_NAME, operation.getDocument());
+        values.put(ETAG_COLUMN_NAME, operation.getEtag());
+        values.put(EXPIRATION_TIME_COLUMN_NAME, now + BaseOptions.DEFAULT_ONE_HOUR); // TODO: how to figure out the next expiration time?
```

Comment:
This is not a release todo, what is that? Do we have a task for it?

Reply:
Oh, this is temporary, a question I wanted to discuss at the next parking lot.